docs/content/test/TestMinimized.md

```diff
@@ -24,3 +24,30 @@ However we could miss some bugs, because debug(not release) sources included, so
 ## Sample
 
 You can check configuration in `:test-app` module.
+
+## If you still have problems 
+
+Keeper can't do all the work here, so if you still can't run test, check these:
+
+### Dynamically referenced resources
+
+You can see in `PageObjectTest`, that we referenced `R.layout.page_object_1`, dynamically creating layout.\
```

Comment:
Enabled shrinkResourses delete resourses referenced only from tests.
Example in PageObjectTest.

Reply:
left as it was, i can hardly parse new variant